src/pages/CommitDetailPage/CommitCoverage/CommitCoverage.jsx

```diff
@@ -45,12 +46,14 @@ function CommitRoutes() {
   const { provider, owner, repo, commit: commitSha } = useParams()
   const { data: tierName } = useTier({ owner, provider })
   const { data: overview } = useRepoOverview({ provider, owner, repo })
-  const { data: commitPageData } = useCommitPageData({
-    provider,
-    owner,
-    repo,
-    commitId: commitSha,
-  })
+  const { data: commitPageData } = useSuspenseQueryV5(
+    CommitPageDataQueryOpts({
+      provider,
+      owner,
+      repo,
+      commitId: commitSha,
```

Comment:
Unrelated question - Do you happen to know how pervasively we allow sha and id to be interchanged like it is here?

Reply:
I don't think it's too bad, but it is a symptom of the API argument for commit sha's being `id`, see:

```graphql
query CommitPageData($owner: String!, $repo: String!, $commitId: String!) {
  owner(username: $owner) {
    isCurrentUserPartOfOrg
    repository(name: $repo) {
      __typename
      ... on Repository {
        private
        bundleAnalysisEnabled
        coverageEnabled
        commit(id: $commitId) { # <- should be sha imo
          commitid
          compareWithParent {
            __typename
          }
          bundleAnalysis {
            bundleAnalysisCompareWithParent {
              __typename
            }
          }
        }
      }
      ... on NotFoundError {
        message
      }
      ... on OwnerNotActivatedError {
        message
      }
    }
  }
}
```

Reply:
Gotcha, yeah I guess the api accepts both as the key. Thanks for looking at it!